pkg/api/plugins.go

```diff
@@ -366,9 +366,11 @@ func (hs *HTTPServer) serveLocalPluginAsset(c *contextmodel.ReqContext, plugin p
 	}
 
 	if hs.Cfg.Env == setting.Dev {
-		c.Resp.Header().Set("Cache-Control", "max-age=0, must-revalidate, no-cache")
+		c.Resp.Header().Set("Cache-Control", "private, max-age=0, must-revalidate, no-cache")
+		c.Resp.Header().Set("X-Grafana-Cache", "true")
 	} else {
-		c.Resp.Header().Set("Cache-Control", "public, max-age=3600")
```

Comment:
@kylebrandt is there a way to introduce these overrides without having to add `private`?

Reply:
Not currently. We would want to make sure it is intentional. Maybe another key to X-Grafana-Cache?

Reply:
Alrighty - just did a quick and dirty change.